exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
```

Comment:
<p>"The views denormalize these tables. So, for instance, instead of the topic name you will see a number in the table.". I think we need to swap around these two sentences. The second one is an explanation of what is stated in the preceding sentence. It that order it's not clear what table you are talking about.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>there is sentence before which introduces these tables</p><p> </p><blockquote>They are built on top of the resulting tables, which contain the data in a normalized form.</blockquote>

Reply:
<p>ah ok, now I get it, thanks.</p>